src/components/SecondaryPanes/CommandBar.css

```diff
@@ -49,31 +49,59 @@ html .command-bar > button:disabled {
   cursor: default;
 }
 
-.command-bar > button > i {
-  height: 100%;
-  width: 100%;
-  display: block;
+img.pause,
+img.stepOver,
+img.stepIn,
+img.stepOut,
+img.resume {
+  background-color: var(--theme-body-color);
 }
 
-.command-bar > button > i > svg {
+.command-bar > button > img {
   width: 16px;
   height: 16px;
+  display: inline-block;
 }
 
-.command-bar button.pause-exceptions {
-  margin-inline-start: 0.5em;
+.command-bar > button > img.pause {
+  mask: url(/images/pause.svg) no-repeat;
 }
 
-.command-bar .subSettings {
-  float: right;
+.command-bar > button > img.stepOver {
+  mask: url(/images/stepOver.svg) no-repeat;
+}
+
+.command-bar > button > img.stepIn {
+  mask: url(/images/stepIn.svg) no-repeat;
+}
+
+.command-bar > button > img.stepOut {
+  mask: url(/images/stepOut.svg) no-repeat;
+}
+
+.command-bar > button > img.resume {
+  mask: url(/images/resume.svg) no-repeat;
 }
 
-.command-bar button.pause-exceptions.uncaught {
-  color: var(--theme-highlight-purple);
+.command-bar > button.pause-exceptions.uncaught.enabled > img.pause-exceptions {
+  background-color: var(--theme-highlight-purple);
 }
 
-.command-bar button.pause-exceptions.all {
-  color: var(--theme-highlight-blue);
+.command-bar > button.pause-exceptions.all > img.pause-exceptions {
+  background-color: var(--theme-highlight-blue) !important;
```

Comment:
why `important`?

Reply:
If `!important` is removed the colour will not change.  I can try moving it to a different location so it cascades properly.  Will give it a try now.

Reply:
yeah... lets try and make it more specific:

`.command-bar > img.pause-exceptions.all `

Reply:
I forgot to add  `.enabled`.  Removed `!important` and now everything works fine.